packages/next/src/server/app-render/create-component-styles-and-scripts.tsx

```diff
@@ -47,19 +47,20 @@ export async function createComponentStylesAndScripts({
             // @ts-ignore
             precedence={precedence}
             crossOrigin={ctx.renderOpts.crossOrigin}
-            key={index}
+            key={`style-${index}`}
           />
         )
       })
     : null
 
   const scripts = jsHrefs
-    ? jsHrefs.map((href) => (
+    ? jsHrefs.map((href, index) => (
         <script
           src={`${ctx.assetPrefix}/_next/${encodeURIPath(
             href
           )}${getAssetQueryString(ctx, true)}`}
           async={true}
+          key={`script-${index}`}
```

Comment:
These end up in https://github.com/vercel/next.js/blob/ea619b624449da310acd253169f4446f9621f85d/packages/next/src/client/components/layout-router.tsx#L472-L476 where they need a key.